python-project-template/.github/workflows/pre-commit-ci.yml.jinja

```diff
@@ -24,7 +24,6 @@ jobs:
       run: |
         sudo apt-get update
         python -m pip install --upgrade pip
-        pip install .
         pip install .[dev]
```

Comment:
I believe we do need `-e` here

Reply:
I'm not sure we need it here, since we're excluding the pytest hook. But please correct me if I'm missing something.